roles/nova-common/templates/etc/nova/nova.conf

```diff
@@ -138,6 +138,10 @@ cpu_mode = host-passthrough
 {% else %}
 cpu_mode = host-model
 {% endif -%}
+live_migration_downtime = 500
+live_migration_downtime_steps = 10
+live_migration_downtime_delay = 75
+live_migration_uri = "qemu+ssh://%s/system?keyfile={{ nova.state_path }}/.ssh/id_rsa"
```

Comment:
Ah, if we make this use `nova@%s` we don't need to set up ssh keys for root. This is preferred, as then we're connecting into a less privileged user.  I'll adjust in a second commit.